matRad_fluenceOptimization.m

```diff
@@ -263,7 +263,7 @@
 backProjection.scenarioProb = pln.multScen.scenProb;
 backProjection.nominalCtScenarios = linIxDIJ_nominalCT;
 
-optiProb = matRad_OptimizationProblem(backProjection);
+optiProb = matRad_OptimizationProblem(backProjection,cst);
```

Comment:
Maybe I would change this to have the cst first, because in the future it will be possible to have multiple projections (multiple quantities)

Reply:
I chose this approach since I was not sure if there are cases where the matRad_OptimizationProblem is called in other places than matRad_fluenceOptimization (and my Pareto optimization). So the idea was to allow "backwards" compatibility for now